### PR TITLE
upgraded cli, fpm and zts to php 7.4

### LINKED ISSUE #44: Support for php 7.4
Do you have any plans to add support for php 7.4?

### PR DESCRIPTION
**Pull request by MariusAmbr**
The other dockerfiles seem to depend on these images, so not sure if we need to add those in a separate PR?

This closes #44

### CODE REVIEW

**Comment by sandrokeil** [comment]
@Duderis Thank you very much for this PR. :+1: 

Can you also add the files for xdebug, blackfire and opcache for cli and fpm, please?

**Comment by MariusAmbr** [comment]
Added the requested files, also added the composer files. So whenever you can, please let us know if we missed anything else @sandrokeil

File: php/inc/php-ext-7.4.m4 [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+RUN set -xe \
+    # workaround for rabbitmq linking issue
+    && ln -s /usr/lib /usr/local/lib64 \
+    # hack to link libgcrypt
+    && ln -s /usr/lib/libgcrypt.so.20 /usr/lib/libgcrypt.so \
+    && ln -s /usr/lib/libgpg-error.so.0 /usr/lib/libgpg-error.so \
+    && apk add --no-cache --virtual .build-deps \
+        $PHPIZE_DEPS \
+    # 7.4 changes https://github.com/php/php-src/blob/PHP-7.4/UPGRADING
```
Comment (sandrokeil) [inline php/inc/php-ext-7.4.m4:52]: Good work! :+1:

**Review by sandrokeil — APPROVED** [review]
Images will be released soon.

**Comment by sandrokeil** [comment]
Just released.

**Pull request merged**